docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: global
+title: INSERT INTO
+displayTitle: INSERT INTO
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `INSERT INTO` statement inserts new rows into a table. The inserted rows can be specified by value expressions, or resulted from a query.
+
+### Syntax
+{% highlight sql %}
+INSERT INTO [TABLE] [db_name.]table_name [partition_spec] value_clause | query
```

Comment:
@huaxingao 
Should it be { value_clause | query } 
or
( value_clause | query )